sql/moz-fx-data-shared-prod/telemetry_derived/feature_usage_v2/query.sql

```diff
@@ -714,6 +748,17 @@ SELECT
     COALESCE(search_with_ads_count_all, CAST(0 AS INT64)),
     NULL
   ) AS search_with_ads_count_all,
+  newtabpage_disabled,
+  IF(
+    '87' < app_version,
```

Comment:
I guess this isn't a new issue, but this check seems likely to fail when we hit Firefox 100? This can be fixed in a followup, but IMO best that we do it soon so this doesn't catch us off guard.